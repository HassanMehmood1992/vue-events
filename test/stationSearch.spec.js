import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Vuex from 'vuex'

import stationSearch from '@/components/stationSearch'

const localVue = createLocalVue()
localVue.use(Vuex)

let wrapper

beforeEach(() => {
    let vuetify = new Vuetify()

    wrapper = mount(stationSearch, {
        store: new Vuex.Store({
            state: { products: [] }
        }),
        localVue,
        vuetify,
        propsData: { label: 'Test Title' },
    })
})

afterEach(() => {
    wrapper.destroy()
})

describe('Station search component', () => {
  
    test('is fully functional', () => {
        expect(wrapper.element).toMatchSnapshot()
    })

})