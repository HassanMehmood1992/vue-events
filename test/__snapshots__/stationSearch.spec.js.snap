// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Station search component is fully functional 1`] = `
<v-autocomplete
  hide-no-data=""
  hide-selected=""
  item-text="name"
  item-value="id"
  items=""
  label="Test Title"
  name="stationSearch"
  no-filter=""
  rules=""
  search-input=""
/>
`;
